lib/integer_palindrome_check.rb

```diff
@@ -0,0 +1,5 @@
+# Returns true if the input positive integer number forms a palindrome. Returns false otherwise.
+def is_palindrome(number)
+  puts "NOT IMPLEMENTED"
+  return true
```

Comment:
How would you feel about doing `raise NotImplementedError` here instead?

Best practice I think is generally to avoid `puts` statements and other things that will print to the console for code being tested, because that output kind of gets randomly interspersed with the test runner's output.

Reply:
Done.